molgenis-app/tomcat-env

```diff
@@ -0,0 +1,12 @@
+environment=production
+opencpu.uri.host=localhost
+elasticsearch.transport.addresses=localhost:9300
+db_uri=jdbc:postgresql://localhost/molgenis
+db_user=molgenis
+db_password=molgenis
+admin.password=admin
```

Comment:
There is a property file in the rpm directory with the same values. Maybe we need to make a choice how to add the properties to the deployment: https://github.com/molgenis/molgenis/blob/master/molgenis-app/rpm/templates/molgenis-server.properties. Or do we need 2?

Reply:
This is intended for configuration of intellij on a dev machine, so you need not manually twiddle with two different lists of env boxes. Sounds like I need a better name and/or move the file around a bit :D

Reply:
Better now?

Reply:
Ah right, to copy you mean! Nice.